handlers/misttriggers/push_end.go

```diff
@@ -143,3 +150,24 @@ func uuidFromPushUrl(uri string) (string, error) {
 	}
 	return path[len(path)-2], nil
 }
+
+func createDtsh(destination string) error {
```

Comment:
Could you add a unit test for this method?

Reply:
Good idea